client-flink/common/src/main/java/org/apache/celeborn/plugin/flink/readclient/CelebornBufferStream.java

```diff
@@ -255,6 +256,12 @@ private void openStreamInternal(@Nullable BiConsumer<Long, Integer> requiredSegm
                 .setInitialCredit(initialCredit)
                 .build()
                 .toByteArray());
+    final SettableFuture<Void> resultFuture;
```

Comment:
Could we create a new method in TransportClient for the sync rpc?

`public ByteBuffer sendRpcSync(ByteBuffer message, RpcResponseCallback callback, long timeoutMs)`